.gitignore

```diff
@@ -11,6 +11,7 @@ ui_*.h
 *.pro.user*
 qrc_*
 docs/html
+.directory
```

Comment:
Ok, then please set this as a global ignore, for example by putting it in ~/.gitignore and doing:

    git config --global core.excludesfile ~/.gitignore